numba/cuda/tests/cudadrv/test_runtime.py

```diff
@@ -17,14 +18,33 @@ def set_visible_devices_and_check(q):
         q.put(-1)
 
 
+if config.ENABLE_CUDASIM:
+    SUPPORTED_VERSIONS = (-1, -1),
+else:
+    SUPPORTED_VERSIONS = ((9, 0), (9, 1), (9, 2),
+                          (10, 0), (10, 1), (10, 2),
+                          (11, 0), (11, 1), (11, 2))
+
```

Comment:
Is it perhaps worth testing the `runtime.supported_versions` property exactly matches this?

Reply:
I felt it wasn't, given what the other tests do.

Reply:
I suppose the other tests are checking both for and against existence in this set of values so it's probably fine :)